packages/inter-protocol/src/clientSupport.js

```diff
@@ -9,8 +9,6 @@ import { parseRatio } from '@agoric/zoe/src/contractSupport/ratio.js';
 const COSMOS_UNIT = 1_000_000n;
 const scaleDecimals = num => BigInt(num * Number(COSMOS_UNIT));
 
-// TODO use '@satisfies" in TS 5.1 to make sure these each conform to OfferMaker interface
-
 // NB: not really a Proposal because the brands are not remotes
 // Instead they're copyRecord like  "{"boardId":"board0257","iface":"Alleged: IST brand"}" to pass through the boardId
 // mustMatch(harden(proposal), ProposalShape);
```

Comment:
This `NB:` is no longer the case, is it? `makeBoardRemote` uses `Far`. The change to smallcaps made it necessary, IIRC.